### PR TITLE
DOCSP-43906: crypt shared embedding

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-43906>
Staging:
- [whats new](https://deploy-preview-575--docs-java.netlify.app/whats-new/#what-s-new-in-5.2)
- [encryption page](https://deploy-preview-575--docs-java.netlify.app/fundamentals/encrypt-fields/)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Comment by rustagir** [comment]
@jyemin @vbabanin tagging you both for this first review, for the other drivers I will not.
On this PR it would also be useful to leave a comment for the compatible/min versions of `mongodb-crypt` for some of the recent driver versions maybe v4.11 onwards? So that I can backport this PR and populate the note with the correct info.

File: source/fundamentals/encrypt-fields.txt [hunk 1]
```diff
@@ -1,3 +1,12 @@
 .. _java-fle:
 
 .. sharedinclude:: dbx/encrypt-fields.rst
+
+   .. replacement:: driver-specific-content
+
+      .. important:: Compatible Encryption Library Version
+
+         The {+driver-short+} uses the `mongodb-crypt
```
Comment (jyemin) [inline source/fundamentals/encrypt-fields.txt:9]: This is good, but let's also be explicit about the full coordinates of the dependency, and how to add it to both Maven and Gradle (like we do [here](https://www.mongodb.com/docs/drivers/java/sync/current/quick-start/#add-mongodb-as-a-dependency)).

**Review by jyemin — CHANGES_REQUESTED** [review]

File: source/fundamentals/encrypt-fields.txt (hunk 1 above)
Comment (rustagir) [reply source/fundamentals/encrypt-fields.txt:9]: Sure, I can add the tabs into this note!

**Review by jyemin — APPROVED** [review]

**Review by norareidy — APPROVED** [review]
LGTM

**Pull request merged**